### ISSUE TITLE
Pointers

### ISSUE THREAD

**Issue by AR786** [rahulkumaran/sudocode#3]
Feature:   Allowing the user to include pointer variables also. 

Solution:
We can have a syntax including the word 'pointer' then the variable name to declare the pointer variable.
Also word 'address' can be used with the variables for the address of the same.
For ex:
initialize pointer int a   (initializes pointer to integer a) 
initialize int b
a= address b     (assigns a the address of b)

Alternative way:
Till now i have only come up with this

Additional context:
The operations on pointers can be done as same in C.
for ex: when we increment the pointer it will increment by value of size of its data type.

**Comment by 7vikpeculiar** [comment]
Can I work on this?

**Comment by rahulkumaran** [comment]
@AR786 , I agree with @UmangGPatel . Let's put this on hold for now because I don't really see how pointers are going to be there in pseudocode. Plus, it's just going to make things way to complicated. For the first release, I guess we're good to go without pointers. Let's see about this if more people request for this.

**Comment by rahulkumaran** [comment]
Well, if you closely observe, I've tried to implement this feature using stacks. So given something like print a, the program tries to figure out whether you want to print a as a string or variable a. If it's the variable a, then it gets the right format specifiers and prints them. Anyway, I think @AviBomb is working on making this more concrete. So yeah, should work.
